### DISCUSSION TITLE
DCE - Malware ping from Webroot

### DISCUSSION THREAD

**Discussion by JPGN** [Tyrrrz/DiscordChatExporter#783] [Q&A]
When attempting to run DCE's non-CLI version, my computer is immediately quarantining it due to stating it includes a system-32-affecting malware. Is this a false positive from Webroot? Please advise.

**Comment by Tyrrrz** [comment]
Yeah, it's because of the bootstrapper used in GUI version which antiviruses seem to be very attracted to. I submitted some false positive reports but they seem to return every other version. If you feel like it, submit a report to their support with the link to the repo, or just ignore it.

**Reply by JPGN** [reply]
Thank you, figured as much since I saw no other hits from searching for reports of malware with DCE.
